ner_v2/detectors/numeral/number_range/en/number_range_detection.py

```diff
@@ -42,8 +42,8 @@ def _custom_num_range_between_num_and_num(self, number_range_list=None, original
         """
         number_range_list = number_range_list or []
         original_list = original_list or []
-        between_range_pattern = re.compile(ur'(between\s+({number}\d+)(?:\s+and|-)'
-                                           ur'\s+({number}\d+))'.format(number=NUMBER_REPLACE_TEXT), re.UNICODE)
+        between_range_pattern = re.compile(r'(between\s+({number}\d+)(?:\s+and|-)'
+                                           r'\s+({number}\d+))'.format(number=NUMBER_REPLACE_TEXT), re.UNICODE)
```

Comment:
Use re.escape

Reply:
Keep u prefix instead of r prefix, some format argument can have unicode literals that can cause error